NOTICE

```diff
@@ -1,5 +1,6 @@
 Druid - a distributed column store.
 Copyright 2012-2015 Metamarkets Group Inc.
+Copyright 2015 Yahoo! Inc.
```

Comment:
I'll check with our counsel and we can update later, I'm fine leaving as is for now.